conda/recipes/libcudf/meta.yaml

```diff
@@ -86,6 +86,7 @@ test:
     - test -f $PREFIX/include/cudf/detail/repeat.hpp
     - test -f $PREFIX/include/cudf/detail/replace.hpp
     - test -f $PREFIX/include/cudf/detail/reshape.hpp
+    - test -f $PREFIX/include/cudf/detail/rolling.hpp
```

Comment:
I wonder why does `conda` want to test (style check?) the header files? They are part of libcudf and should have been tested by CI tests before merging.

Reply:
This is not a unit test. This is a file existence test, using the Unix command-line tool `test`. Type `man test`.

Reply:
This ensures that our public headers are where they are expected to be.